app/src/main/java/com/wizeline/mobilenews/domain/usecases/GetCommunityNewsUseCaseImpl.kt

```diff
@@ -9,12 +14,22 @@ import javax.inject.Inject
 class GetCommunityNewsUseCaseImpl @Inject constructor(
     private val communityRepository: CommunityRepository
 ) : GetCommunityNewsUseCase, NewsDataToNewsDomain<CommunityArticle, Article>() {
-    override suspend fun invoke(
-        query: List<String>,
-        dateFrom: Long,
-        dateTo: Long?,
-        pageSize: Int,
-        page: Int
-    ): List<CommunityArticle> =
-        communityRepository.searchNews(query, dateFrom, dateTo, pageSize, page)
+//    override suspend fun invoke(
```

Comment:
Remove this as well, if we need it later we can get it through the tree history.